client/app/common/models/Environment.js

```diff
@@ -10,7 +10,8 @@ angular.module('EnvironmentManager.common').factory('Environment',
 
     Environment.all = function (params) {
       var query = params && params.query;
-      return $http.get('/api/v1/config/environments', { params: query, cache: true }).then(function (response) {
+      var useCache = !params || (params.useCache !== false);
```

Comment:
`params === undefined || ...`?
it makes intention more explicit and avoids all weird JS casting issues (0 == false), but maybe I'm just fussy :)